natives.json

```diff
@@ -10489,8 +10473,8 @@
       "comment": "",
       "params": [
         {
-          "type": "Any",
-          "name": "p0"
+          "type": "const char*",
+          "name": "music_name"
```

Comment:
Parameters should not include underscores. Please update the code to reflect this convention. Parameter names should use camelCase.